### PR TITLE
Views cache ComponentPools, bug fixes, and explicit template instantiation for faster compilation

### PR DESCRIPTION
**Pull request by Chris-plusplus**

### CODE REVIEW

File: archimedes_bin/main.cpp
[the remark this answers is not in the document]
Comment (Chris-plusplus) [reply archimedes_bin/main.cpp:1]: nah, they're my manual tests, I'll remove them once everything is ready to be merged

File: conanfile.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ def requirements(self):
         self.requires("gtest/1.13.0")
         self.requires("stb/cci.20230920", override=True)
         self.requires("draco/1.5.6", override=True)
+        self.requires("entt/3.13.2")
```
[the remark this answers is not in the document]
Comment (Chris-plusplus) [reply conanfile.py:17]: situation same as main.cpp

File: archimedes_bin/main.cpp
[the remark this answers is not in the document]
Comment (Twarug) [reply archimedes_bin/main.cpp:1]: If so, they shouldn't be commit then.

[the remark this answers is not in the document]
Comment (Chris-plusplus) [reply archimedes_bin/main.cpp:1]: i needed them to check if they compile on my phone

[the remark this answers is not in the document]
Comment (Twarug) [reply archimedes_bin/main.cpp:1]: Still they shouldn't be in PR

**Review by Twarug — APPROVED** [review]

**Pull request merged**